hydra/test_utils/launcher_common_tests.py

```diff
@@ -431,6 +437,17 @@ def get_test_app_working_dir(self) -> Optional[Path]:
         """
         return None
 
+    def generate_custom_cmd(self) -> Callable:
```

Comment:
would it be possible to override `get_test_app_working_dir ` instead? 

it seems like this function is created for modify the working dir as well.

Reply:
No, this is different.
It's not the working dir that is different, it's the command we use to execute the generated code.